specs/gloas/fork-choice.md

```diff
@@ -278,12 +279,12 @@ def is_supporting_vote(store: Store, node: ForkChoiceNode, message: LatestMessag
     if node.root == message.root:
         if node.payload_status == PAYLOAD_STATUS_PENDING:
             return True
-        if message.slot <= block.slot:
-            return False
-        if message.payload_present:
-            return node.payload_status == PAYLOAD_STATUS_FULL
-        else:
-            return node.payload_status == PAYLOAD_STATUS_EMPTY
+        if message.slot > block.slot:
```

Comment:
Would love to get some input on this change. I had trouble wrapping my head around why we would return `False` when `message.slot <= block.slot`, and I think this change reads more natural to me.

Reply:
Umm.. It reads semantically the same to me. I personally prefer the previous one as it doesn't have a nested if-statement. It's not part of this PR but I'd love to delete the whitespace at L288 though.

This is how I understand returning `False` when `message.slot <= block.slot`:

1) It returns `False` when `message.slot < block.slot` because `block` is not part of the chain that the `message` supports. `block` is may or may not a descendent of the supported chain.

2) It returns `False` when `message.slot == block.slot` because the `message` should support `PAYLOAD_STATUS_PENDING` path, not `PAYLOAD_STATUS_FULL` nor `PAYLOAD_STATUS_EMPTY`.